plonky/src/api.rs

```diff
@@ -226,3 +228,33 @@ pub fn generate_aggregation_verifier(
     verifier::recursive_plonk_verifier::create_verifier_contract_from_default_template(config, sol);
     Result::Ok(())
 }
+
+pub fn groth16_setup(circuit_file: &String, witness: &String, curve_type: &str) -> Result<()> {
```

Comment:
Does the setup function output json format? Should the proving key and verification key be separated or should they be put together?

Reply:
Be separated. VK will be distributed to the verifier.